rust/kernel/user_ptr.rs

```diff
@@ -46,6 +46,31 @@ unsafe impl ReadableFromBytes for i32 {}
 unsafe impl ReadableFromBytes for i64 {}
 unsafe impl ReadableFromBytes for isize {}
 
+/// Specifies that a type can be safely writable to byte slices.
+///
+/// This means that we don't read undefined values when reading the memory contents (which leads to
+/// UB). It also ensures that no potentially sensitive information is leaked into the byte slices.
```

Comment:
Should all the "reads" in this paragraph be "writes"?

Reply:
I meant reading the memory contents (before writing). I updated the comment, hopefully it's clearer now. PTAL.